lib/index.js

```diff
@@ -7,9 +7,10 @@ const spawn = require('cross-spawn');
 const path = require('path');
 const tildify = require('tildify');
 const Spinner = require('cli-spinner').Spinner;
+const readline = require('readline');
+const createReactApp = require.resolve('create-react-app');
```

Comment:
can we move it to the function? It's not actual require of a module, I'd put it next to args so we don't mix things